providers/standard/src/airflow/providers/standard/sensors/time.py

```diff
@@ -100,32 +77,61 @@ def __init__(
         self,
         *,
         target_time: datetime.time,
+        deferrable: bool = False,
```

Comment:
I think it should be

```suggestion
        deferrable: bool = conf.getboolean("operators", "default_deferrable", fallback=False)
```

not sure why the pre-commit is not working, I'll take a look afterward

Reply:
Where is the pre-commit not working?

Reply:
https://github.com/apache/airflow/pull/50934

Reply:
Fixed in this PR